src/extensions/cms/store.js

```diff
@@ -0,0 +1,59 @@
+import config from 'config'
+import fetch from 'isomorphic-fetch'
+
+const state = {
+  cmsPages: [],
+  cmsBlock: []
+}
+
+const getters = {
+  getBlock: (state) => (id) => {
+    return state.cmsBlock.find(item => item.id === id)
```

Comment:
Could You add the in-browser cache for the static content to improve the performance? we can use kind of localForage sync like we have in the shopping cart https://github.com/DivanteLtd/vue-storefront/blob/6be64960d9dc5e728fcdda75068f80d8dbc8436a/core/store/index.js#L59

Or, explicitly call `setItem` and `getItem` on some newly created localForage collection (please add the localForage collection in here: https://github.com/DivanteLtd/vue-storefront/blob/cf49b4c85bb562a55e475d1d7ee33c3e36ab8dfd/core/store/index.js#L143

Reply:
added